### ISSUE TITLE
Just thanks for the awesome plugin. This is not an issue.

### ISSUE THREAD

**Issue by avioli** [jisaacks/GitGutter#539]

**Comment by gundamew** [comment]
Me too. Thank you @jisaacks for the awesome plugin. 👍

**Comment by SonGokussj4** [comment]
As I read: "seems to become deprecated / obsolete for ST3200+ in its current form" I disagree. Your plugin is awesome. So much functionality (that I'm not even using), really subtle, you can say... sublime :-) 
Thank you for this plugin.

**Comment by narikazu** [comment]
Thank you very much @jisaacks for the greatest plugin! I really appreciate that you've released the new versions of this plugin often.

**Comment by godbout** [comment]
Was coming here to do the same 😂️

Great work!

**Comment by shadycuz** [comment]
I agree with @SonGokussj4 and came here to say the same. They added similar features but they didn't git them all 😉 . The best thing about your plugin in the ability to right-click a change and undo it. I also like that the preview of what changed appears when I hover my mouse over the gutter. 

Thanks for the awesome plugin.

**Comment by jisaacks** [comment]
I appreciate all of the gratitude but lately it is @deathaxe that deserves all the credit. He has been doing a great job! Thank you very much @deathaxe.

**Comment by deathaxe** [comment]
Thank you very much. 

Guess I need to clarify the following statement:

> "seems to become deprecated / obsolete for ST3200+ in its current form"

It does not mean, we don't need to continue with GitGutter. As some of GitGutter's basic features were moved to ST core, I just wonder whether GitGutter could be made a bit lightert to avoid duplicated work and reduce CPU usage a bit (if it even matters).

ST's mini diff does a good job. My hope during the last dev cycle was to be able to leave all the diffing while typing to ST as it is much quicker and cheaper as calling `git diff` all the time.

GitGutter could then concentrate on the additional features, like line annotations, diff popup, status messages etc. Unfortunatelly ST does not yet add diff markers to the minimap, wich I don't want to loose. So any bigger changes would need to wait anyway.

**Comment by lode** [comment]
I also want to mention that this plugin is great. I disabled sublime's new diff and am using this plugin again. It does a better job in my eyes (and _to_ my eyes, as you've seen in https://forum.sublimetext.com/t/how-do-i-hide-icons-show-git-status/42363).

**Comment by avioli** [comment]
@deathaxe Thank you, too.

**Comment by Giovancruz** [comment]
I came here to see if is possible to deactivate the behavior of ST3 Git, not found, @lode how you do that? 

BTW, Tks for the awesome plugin.

**Comment by deathaxe** [comment]
@Giovancruz The settings you might be interested in are:

```
"show_git_status": false,
"mini_diff": false
```

_To be added to Preferences.sublime-settings._

**Comment by rudiedirkx** [comment]
I love this! I came here just for this reason! Much better than ST3's built-in GIT stuff.

**Comment by arongile** [comment]
Seriously, thank you!!! Love it.
